### PR TITLE
Fix unexpected override

### PR DESCRIPTION
**Pull request by waitingallday**
Occurs on search results in fixed header

### CODE REVIEW

File: assets/targets/injection/header/index.scss [hunk 1]
```diff
@@ -78,11 +78,15 @@
           }
         }
       }
-      
+
       a > span {
```
Comment (axelboc) [inline assets/targets/injection/header/index.scss]: How about `a > span:not([data-icon])` here? This is specifically for cases where the breadcrumb contains an inner span that wraps both the icon and the label.

Comment (waitingallday) [inline assets/targets/injection/header/index.scss]: Always a little wary of using `:not` for compatibility with legacy browsers, but this specific use (svg icon in the fixed header) won't work on legacy browsers anyway so :+1:

Comment (axelboc) [inline assets/targets/injection/header/index.scss]: Yeah and the mis-alignment it fixes is very minor in the first place.

**Comment by waitingallday** [comment]
Just realised why it came up - this one only applies to the default header when no breadcrumbs are defined. Might need to change the way this is done so it's consistent with the provided breadcrumb solution.

**Pull request merged**